File: app/src/main/java/org/stepic/droid/events/submissions/SuccessGettingLastSubmissionEvent.java
package org.stepic.droid.events.submissions;

import org.stepic.droid.model.Submission;

public class SuccessGettingLastSubmissionEvent extends SubmissionBase {
    private final Submission mSubmission;

    public SuccessGettingLastSubmissionEvent(long attemptId, Submission submission) {
        super(attemptId);
        mSubmission = submission;
    }

    public Submission getSubmission() {
        return mSubmission;
    }
}

File: app/src/main/java/org/stepic/droid/events/submissions/SuccessGettingLastSubmissionEvent.kt
package org.stepic.droid.events.submissions

import org.stepic.droid.model.Submission

class SuccessGettingLastSubmissionEvent @JvmOverloads constructor(
        attemptId: Long,
        val submission: Submission? = null,
        val numberOfSubmissionsOnFirstPage: Int = 0) : SubmissionBase(attemptId)